universal-login-relayer/lib/core/services/validators/GasValidator.ts

```diff
@@ -1,15 +1,16 @@
-import {estimateGasDataFromSignedMessage} from '@universal-login/contracts';
+import {estimateGasDataFromUnsignedMessage} from '@universal-login/contracts';
 import {SignedMessage, ensure} from '@universal-login/commons';
 import {InsufficientGas} from '../../utils/errors';
 import IMessageValidator from './IMessageValidator';
 
-export const GAS_BASE = 0;
+export const GAS_BASE = 21000;
```

Comment:
The gas base seems to be too low, minimum transaction gas without data is: 102487 (let's use 105000).
https://circleci.com/gh/UniversalLogin/UniversalLoginSDK/20682 (see 'Ether transfer')